### ISSUE TITLE
Json-Field doesn't work with Panels

### ISSUE THREAD

**Issue by Brotzka** [armincms/json#3]
you can simplify your code like following:

```` 
         new Panel('Fahrzeug-Provider-Details',  Json::make('vdp_data', [
                    Select::make('Aktualisierungsintervall', 'update_interval')->options([
                        'everyMinute'           => 'Jede Minute',
                        'everyFiveMinutes'      => 'Alle 5 Minuten',
                        'everyTenMinutes'       => 'Alle 10 Minuten',
                        'everyFifteenMinutes'   => 'Alle 15 Minuten',
                        'everyThirtyMinutes'    => 'Alle 30 Minuten',
                        'hourly'                => 'Stündlich',
                        'daily'                 => 'Täglich',
                        'weekly'                => 'Wöchentlich'
                    ]),

                    Text::make('Kundennummer', 'value'), 
            ])->hideFromIndex()->toArray()),

````

**Comment by zareismail** [comment]
Hey.
Thanks for reporting.
It will be resolved soon!

**Comment by bobbybouwmann** [comment]
@zareismail You have an update on this issue? I have the same issue with the Panel.

**Comment by zareismail** [comment]
Hi. Thanks for reporting.
resolved in release 0.5.1

**Comment by zareismail** [comment]
> @zareismail You have an update on this issue? I have the same issue with the Panel.

resolved in release 0.5.1

**Comment by bobbybouwmann** [comment]
@zareismail You're fast! Awesome work man!
